### PR TITLE
SAK-45171 Add enrollment status translations to UI

### PR DESCRIPTION
**Pull request by adrianfish**
https://jira.sakaiproject.org/browse/SAK-45171

### CODE REVIEW

File: library/src/morpheus-master/sass/modules/tool/sitemanage/_sitemanage.scss [hunk 1]
```diff
@@ -678,6 +678,10 @@
     }
   }
 
+  .enrollmentStatus {
```
Comment (ottenhoff) [inline library/src/morpheus-master/sass/modules/tool/sitemanage/_sitemanage.scss:681]: just use bootstrap .small instead of adding new class?

Comment (adrianfish) [reply library/src/morpheus-master/sass/modules/tool/sitemanage/_sitemanage.scss:681]: Aren't we trying to move away from using bootstrap, steadily?

Comment (adrianfish) [reply library/src/morpheus-master/sass/modules/tool/sitemanage/_sitemanage.scss:681]: Shawn always gives me the impression that bootstrap is bad down the road.

Comment (ottenhoff) [reply library/src/morpheus-master/sass/modules/tool/sitemanage/_sitemanage.scss:681]: no idea 🤷

Comment (adrianfish) [reply library/src/morpheus-master/sass/modules/tool/sitemanage/_sitemanage.scss]: I'm pretty sure that's the case, just trying to move away from our dependence on bootstrap for basic styling stuff. I've updated that class to use one of our variables.

**Review by ottenhoff — APPROVED** [review]

**Pull request merged**